buildout.cfg

```diff
@@ -1,9 +1,48 @@
 [buildout]
 
-# use this extend one of the buildout configuration:
 extends =
-# -*- mrbob: extra extends -*-
-#    test_plone43.cfg
-#    test_plone50.cfg
-#    test_plone51.cfg
-    test_plone52.cfg
+    https://raw.githubusercontent.com/collective/buildout.plonetest/master/plone-5.2.x.cfg
+    base.cfg
+
+eggs +=
+    Products.PDBDebugmode
+    pdbpp
+
+[versions]
+# Version 4 targets Plone 6 and newer only
+# collective.easyform = < 4.0.0
+
+# Buildout was not able to install the following.
+# I had to manyally pip install them into the virtualenv
+cachetools = 3.1.1
+cssutils = 1.0.2
+suds = 0.4
+
+collective.easyform = 3.2.0
```

Comment:
will this not limit us getting the benefits from 3.x branch
I see Mr. Tango made some semi-recent fixes?
https://github.com/collective/collective.easyform/commits/3.x